### DISCUSSION TITLE
set relevance equation for subquestions

### DISCUSSION THREAD

**Discussion by Deusdeorum** [edgarrmondragon/citric#971] [Q&A] [answered]
Great package! Do you know if it is possible to add "relevance equation" for subquestions?

Suppose we have a "multiple choice" question in a survey, with some subquestions. Using `set_question_properties` would it be possible to set the relevane equation for those subquestions?

**Comment by edgarrmondragon** [comment, marked as answer]
Hi @Deusdeorum!

Yes, each subquestion has its own question ID that you can retrieve with `client.get_question_properties(<parent question ID>)`:

```python
>>> for qid, sub in client.get_question_properties(176)["subquestions"].items():
...     print(qid, sub["title"], sub["question"])
188 SQ003 Object Relational Mappers (SQLAlchemy, etc.)
187 SQ002 Web frameworks (FastAPI, Flask, Django, etc.)
186 SQ001 Data manipulation libraries (Pandas, Numpy, Polars, etc.)
```

You can then call `set_question_properties` with each of the subquestions:

```python
client.set_question_properties(186, relevance='< your custom relevance equation >')
```

**Reply by Deusdeorum** [reply]
Ah, of course! As simple as that - thanks.
